wayland-client/src/conn.rs

```diff
@@ -217,8 +217,9 @@ impl<'a> HandleInner<'a> {
 }
 
 impl<'a> ConnectionHandle<'a> {
-    pub(crate) fn from_handle(handle: &'a mut Handle) -> ConnectionHandle<'a> {
-        ConnectionHandle { inner: HandleInner::Handle(handle) }
+    /// Returns the underlying [`Handle`] from `wayland-backend`.
+    pub fn handle(&mut self) -> &mut Handle {
```

Comment:
Let's name this `backend_handle()` to be explicit and not confuse it with the `.handle()` method on `Connection`